addons/api/addon-test-support/handlers/client-daemon-search.js

```diff
@@ -47,7 +47,7 @@ export default function setupStubs(hooks) {
           .withArgs('searchClientDaemon')
           .onCall(i)
           .returns({
-            [type]: this.server.db[type].map((model) => ({
+            [type]: this.server.db[type]?.map((model) => ({
```

Comment:
Added this for tests since there's a scenario that I pass in an empty string in place of a `sessions` string to get back no results since the goal was to test canceling a session. Let me know if you know of a better way to do this. (ref ln 29 in index-test.js)